### DISCUSSION TITLE
Login takes a very long time to load

### DISCUSSION THREAD

**Discussion by Forceu** [authelia/authelia#6027] [Troubleshooting] [answered]
### Version

v4.37.5

### Deployment Method

Docker

### Reverse Proxy

NGINX

### Reverse Proxy Version

nginx/1.22.1

### Description

Possibly related to #1707 , might however also be a problem related to nginx

The screen to login takes more than 2 minutes to load and after entering the credentials another 2. This happens every time on multiple browsers / OS. I am using podman and nginx for running authelia, but the same problem occurs for running authelia bare metal. If enter the CSS script url manually, it loads immediately.

![image](https://github.com/authelia/authelia/assets/1593467/a50e4da2-353a-45e1-9b47-4c28f57c6a42)

### Reproduction

Loading the login page

### Configuration (Authelia)

```yaml
server:
  host: 0.0.0.0
  port: 9091
log:
  level: info
default_redirection_url: https://XXXXXXX
totp:
  issuer: authelia.com
authentication_backend:
  file:
    path: /home/authelia/config/authelia/users_database.yml
    watch: false
    search:
      email: false
      case_insensitive: true
    password:
      algorithm: sha512
      iterations: 1000
      key_length: 32
      salt_length: 16
      memory: 1024
      parallelism: 8
```

### Build Information

```shell
Last Tag: v4.37.5
State: tagged clean
Branch: v4.37.5
Commit: 566a0d7fc71b450123ad33d350cd3890d311da82
Build Number: 17068
Build OS: linux
Build Arch: amd64
Build Date: Wed, 21 Dec 2022 19:54:54 +1100
Extra:
```

### Logs (Authelia)

```shell
INFO[0000] Authelia v4.37.5 is starting
INFO[0000] Log severity set to info
INFO[0000] Storage schema is being checked for updates
INFO[0000] Storage schema is already up to date
INFO[0000] Initializing server for non-TLS connections on '[::]:9091' path '/'
```

### Logs (Proxy / Application)

```shell
2023/09/24 15:51:03 [error] 4075044#4075044: *10 connect() failed (110: Connection timed out) while connecting to upstream, client: 192.168.2.3, server: auth.XXXXX, request: "GET /static/css/index.40feef90.css HTTP/2.0", upstream: "http://[::1]:9091/static/css/index.40feef90.css", host: "auth.XXXXX", referrer: "https://auth.XXXXXXXX/2fa/webauthn"
```

### Documentation

Nginx config:

```
server {
        listen 443 ssl;
        listen [::]:443 ssl;
        ssl_certificate /home/acme/certs/cert.pem;
        ssl_certificate_key /home/acme/certs/key.pem;       

	proxy_buffering off;

        server_name auth.XXXXXXXXXX;

	 location / {
		proxy_pass http://localhost:9091;
		include config/homelab.local/proxy.conf;
	}

}
```
                                                                               
```
client_body_buffer_size 128k;

#Timeout if the real server is dead
proxy_next_upstream error timeout invalid_header http_500 http_502 http_503;

# Advanced Proxy Config
send_timeout 5m;
proxy_read_timeout 360;
proxy_send_timeout 360;
proxy_connect_timeout 360;

# Basic Proxy Config
proxy_set_header Host $host;
proxy_set_header X-Real-IP $remote_addr;
proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
proxy_set_header X-Forwarded-Proto $scheme;
proxy_set_header X-Forwarded-Host $http_host;
proxy_set_header X-Forwarded-Uri $request_uri;
proxy_set_header X-Forwarded-Ssl on;
proxy_redirect  http://  $scheme://;
proxy_http_version 1.1;
proxy_set_header Connection "";
proxy_cache_bypass $cookie_session;
proxy_no_cache $cookie_session;
proxy_buffers 64 256k;

# If behind reverse proxy, forwards the correct IP
set_real_ip_from 10.0.0.0/8;
set_real_ip_from 172.16.0.0/12;
set_real_ip_from 192.168.0.0/16;
set_real_ip_from fc00::/7;
real_ip_header X-Forwarded-For;
real_ip_recursive on;




```

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)

- [X] This is a bug report and not a support request

- [X] I have read the security policy and this bug report is not a security issue or security related issue

- [X] I have either included the complete configuration file or I am sure it's unrelated to the configuration

- [X] I have either included the complete debug / trace logs or the output of the build-info command if the logs are not relevant

- [X] I have provided all of the required information in full with the only alteration being reasonable sanitization in accordance with the [Troubleshooting Sanitization](https://www.authelia.com/r/sanitize) reference guide

- [X] I have checked for related proxy or application logs and included them if available

- [X] I have checked for related issues and checked the documentation

**Comment by james-d-elliott** [comment, marked as answer]
You probably don't want to be using `localhost` to begin with. Does the same problem occur with `proxy_pass http://localhost:9091;`? Also this looks like a support issue not a bug as we do integration testing with chrome webdriver which would experience this issue if it was an issue with Authelia itself rather than a proxy configuration or infrastructure issue, so I'm converting it. We can convert it back later if it tuns out I'm wrong.

**Reply by james-d-elliott** [reply]
Might be worth showing the complete trace logs for the actions you're describing, and upload a [HAR](https://www.authelia.com/reference/guides/http-archive-file/). Also can you describe what you mean by bare metal?

**Reply by Forceu** [reply]
Changing it from localhost to 127.0.0.1 worked for some reason. Very interesting, thanks a lot for your help!

**Reply by james-d-elliott** [reply]
Yeah only thing I can think of is weird IPv6 or resolution issues.
